src/Board.py

```diff
@@ -50,6 +51,85 @@ def __init__(self, style: str = "default", size: int = 5) -> None:
         else:
             raise ValueError("Unrecognised map style")
 
+    def get(self, coords):
```

Comment:
It would be good to have typing information for this function (P.S. do you have `strict` mode enabled in your vscode? It should complain about missing typing info here if so).

I'd suggest defining a type early in the file:
```python3
Coords = Tuple[int, int]
```

```suggestion
    def get(self, coords: Coords):
```
(this would require importing `Tuple` from `typing` too)